07-errors.md

```diff
@@ -0,0 +1,301 @@
+# Error Conditions
+
+This document specifies the error conditions in the FVM and the associated error codes/numbers.
+
+## Exit Codes
+
+Exit codes are numeric, 32bit integer codes that are returned when an actor exits both successfully and in error.
+Exit codes appear on-chain inside the message receipt, and therefore form part of consensus.
+
+We classify exit codes as "normal" and "abnormal".
+
+The only _normal_ exit code is `0`, or `Ok`. The `Ok` exit code indicates that the message was
+applied successfully.
+
+All other exit codes are _abnormal_ and indicate that the message _was not_ applied and that the
```

Comment:
I would prefer to simply talk about **non-zero** codes. Abnormal is a relative and subjective term (e.g. I consider an out of gas to be a _normal_ circumstance, for example, but would consider a call after a deletion to be abnormal).

Reply:
I'll try that, it may be enough.

Reply:
Yeah, that's better.